src/components/Markdown.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+  import marked from "marked";
+
+  let markdown;
+  export let text = "";
+</script>
+
+{#if markdown}
+  {@html marked.parseInline(markdown.textContent)}
+{/if}
+<div class="raw-markdown hidden" bind:this={markdown}>
+  <slot>{text}</slot>
+</div>
```

Comment:
Hmm, it's a bit unfortunate that this adds a hidden div to the document:

![image](https://user-images.githubusercontent.com/20569/97752546-8f9bcb00-1aca-11eb-8c9c-c5b01a52c428.png)

I offhand don't know of a better way of doing this though, and your solution does work well, so am inclined to merge.